src/components/_cards/WithdrawQueueCard.tsx

```
@@ -140,27 +154,57 @@ const WithdrawQueueCard = (props: TableProps) => {
+          request.creationTime,
+          request.secondsToMaturity,
+          request.secondsToDeadline,
+        ]
+        const gasLimitEstimated = await estimateGasLimitWithRetry(
+          boringQueue?.estimateGas.cancelOnChainWithdraw,
+          boringQueue?.simulate.cancelOnChainWithdraw,
+          [withdrawTouple],
+          330000,
+          address
+        )
+        // @ts-ignore
```

ESLint

[error] Do not use "@ts-ignore" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)